site/content/extend/plugins/webapp/actions.md

```diff
@@ -1,115 +1,502 @@
 ---
-title: Redux Actions
-heading: "Redux Actions for Web App Plugins"
+title: Interactions with Redux
+heading: "Redux Interactions for Web App Plugins"
 description: "Mattermost-redux is a library of shared code between Mattermost JavaScript clients. Learn how to use Redux actions with a plugin."
 date: 2018-07-10T00:00:00-05:00
 weight: 11
 ---
 
-When building web app plugins, it is common to perform actions that web and mobile apps already support. The majority of these actions exist in [mattermost-redux](https://github.com/mattermost/mattermost-redux), our library of shared code between Mattermost JavaScript clients.
+When building web app plugins, it is common to perform actions that web and mobile apps already support. The majority of these actions exist in [mattermost-redux](https://github.com/mattermost/mattermost-redux), our library of shared code between Mattermost JavaScript clients. The `mattermost-redux` library exports types and functions that are imported by the web application and mobile application. These functions can be imported by plugins, and used the same way. There are a few different kinds of functions exported by the library:
+
+[actions](https://github.com/mattermost/mattermost-redux/tree/master/src/actions) - Actions perform API requests and can change the state of Mattermost.
+
+[client](https://github.com/mattermost/mattermost-redux/tree/master/src/client) - The client package can be used to instantiate a Client4 object, to interact with the Mattermost API directly. This is useful in plugins as well as javascript server applications communicating with Mattermost.
+
+[constants](https://github.com/mattermost/mattermost-redux/tree/master/src/constants) - An assortment of constants within Mattermost's data model
+
+[selectors](https://github.com/mattermost/mattermost-redux/tree/master/src/selectors) - Selectors return certain data from the Redux store, such as getPost which allows you get a post by id from the Redux store.
+
+[store](https://github.com/mattermost/mattermost-redux/tree/master/src/store) - Functions related to the Redux store itself
+
+[types](https://github.com/mattermost/mattermost-redux/tree/master/src/types) - Various types of objects in Mattermost's data model. This is useful in plugins as well as javascript server applications communicating with Mattermost.
```

Comment:
```suggestion
[types](https://github.com/mattermost/mattermost-redux/tree/master/src/types) - Various types of objects in Mattermost's data model. These is useful in plugins as well as TypeScript server applications communicating with Mattermost.
```

Reply:
That will be 'These are' then. Committing that right away, Thanks.